source/_posts/2020-11-16-short-python-screencasts-for-$50-slash-year.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Short Python screencasts for $50/year"
+date: 2020-11-16 15:26:13 -0800
+comments: true
+categories: python
+---
+
+I'm really excited to announce that Python Morsels subscriptions will now include concise screencasts.
+
+I will now be publishing at least one new Python screencast to active subscribers every week.
+
+These will be short, single-topic screencasts that won't waste your time.
+Each screencast will be at most 5 minutes in length and each will be as succinct as I can make it.
```

Comment:
Instead of _that won't waste your time_ can we say like _that will be helpful in understanding a concept_ ( I think watching a screencast actually introduces you to a concept or updates you)